File: 프로그래머스/1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let result = '';

    for (let i = 0; i < s.length; i++) {
        let x = 0;

        if ('a' <= s[i] && s[i] <= 'z') {
            x = s[i].charCodeAt(0) + n;
            if (x > 'z'.charCodeAt(0)) {
                x -= 26;
            }
            result += String.fromCharCode(x);
        } else if ('A' <= s[i] && s[i] <= 'Z') {
            x = s[i].charCodeAt(0) + n;
            if (x > 'Z'.charCodeAt(0)) {
                x -= 26;
            }
            result += String.fromCharCode(x);
        } else {
            result += s[i];
        }
    }

    return result;
}

File: 프로그래머스/1/12926. 시저 암호/시저 암호.py
def solution(s, n):
    
    result = ''
    for i in range(len(s)):
        x=0
        
        if ord('a')<=ord(s[i])<=ord('z'):
            x=ord(s[i])+n
            if x>ord('z'):
                x-=26
            
            result+=chr(x)
            
            
            
        
        elif ord('A')<=ord(s[i])<=ord('Z'):
            x=ord(s[i])+n
            if x>ord('Z'):
                x-=26
            
            result+=chr(x)
        
        else:
            result +=s[i]
    
    
    
    return result